app/jobs/set_collection_job.rb

```diff
@@ -54,8 +54,8 @@ def set_collection_and_index_safely(current_druid, log)
   def check_can_set_collection!(cocina, state_service)
     raise "#{cocina.externalIdentifier} is not open for modification" unless state_service.allows_modification?
 
-    new_collection_ids.each do |new_collection_id|
-      raise "user not authorized to move #{cocina.externalIdentifier} to #{new_collection_ids}" unless ability.can?(:manage_item, cocina, new_collection_id)
+    new_collection_ids.each do |new_collection_ids|
```

Comment:
Please don't shadow the method call with this variable name. Shouldn't this be singular anyways?

Reply:
Oh, good catch, I didn't realize a global replace hit those lines. Fixed.